src/Auth0/Login/Traits/Auth0UserTrait.php

```diff
@@ -0,0 +1,112 @@
+<?php
```

Comment:
Here's the trait we were discussing. Is this enough to be helpful on other user models?